.github/ISSUE_TEMPLATE/add-new-platform.md

```diff
@@ -0,0 +1,42 @@
+# Adding a new supported platform(N)
+
+## At Development
+
+- [ ] Create a WIP draft pull request on CoreOS-Assembler with new platform support [here](https://github.com/coreos/coreos-assembler)
+- [ ] Create a WIP draft pull request on the [fedora-web](https://pagure.io/fedora-web/websites) project
+    - [ ] `Add platform to the sites/static/js/coreos-download.js`
+- [ ] Create a WIP draft pull request on the [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
```

Comment:
Platforms with cloud images also need an `Images` field, and a corresponding PR to stream-metadata-rust.

The metadata docs in fedora-coreos-tracker also need updating.

Reply:
```suggestion
- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
    - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
    - [ ] `Add platform to the Media struct in release/release.go`
    - [ ] `Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go` 
    - [ ] (Cloud Only) `Cloud Images need to have a Images field`
- [ ] (Cloud Only) [stream-metadata-rust](https://github.com/coreos/stream-metadata-rust/)
    - [Example PR](ERM)
```

So I was thinking something like this. However, I am not finding a good example of a pr for the stream-metadata-rust adding a new platform. Or I am and just not recognizing it. 

Do you have any recommended Pr's for this @bgilbert ?

Reply:
Thats perfect thank you @miabbott !